lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -908,7 +908,7 @@ def stop_job(self, job_wrapper):
                 self.__cleanup_k8s_job(k8s_job)
             # TODO assert whether job parallelism == 0
             # assert not job_to_delete.exists(), "Could not delete job,"+job.job_runner_external_id+" it still exists"
-            log.debug(f"({job.id}/{job.job_runner_external_id}) Terminated at user's request")
+            log.debug("(%s/%s) Terminated at user's request", job.id, job.job_runner_external_id)
 
         except Exception as e:
             log.exception(
```

Comment:
This `log.exception()` call also uses `str.format()`, but for some reason it hasn't been rewritten.